code/site/components/com_pages/model/filesystem.php

```diff
@@ -11,17 +11,21 @@ class ComPagesModelFilesystem extends ComPagesModelCollection
 {
     protected $_path;
 
+    protected $_base_path;
+
     public function __construct(KObjectConfig $config)
     {
         parent::__construct($config);
 
-        $this->_path = $config->path;
+        $this->_path      = $config->path;
+        $this->_base_path = $config->base_path;
     }
 
     protected function _initialize(KObjectConfig $config)
     {
         $config->append([
-            'path'  => '',
+            'path'      => '',
+            'base_path' =>  Koowa::getInstance()->getRootPath().'/joomlatools-pages',
```

Comment:
@johanjanssens Should we make this path a constant? I think we duplicated `joomlatools-pages` in a few places

Reply:
Fair point, added a ticket: https://github.com/joomlatools/joomlatools-pages/issues/162